reana_workflow_controller/rest/workflows.py

```diff
@@ -252,7 +262,7 @@ def get_workflows(paginate=None):  # noqa
                 "human_readable": "",
                 "raw": -1,
             }
-            if verbose:
+            if include_workspace_size or verbose:
```

Comment:
I think here `or verbose` is not needed due to how you get `include_workspace_size` value.

```
include_workspace_size = request.args.get("include_workspace_size", verbose)
```

- If `--include-workspace-size` is passed, `include_workspace_size` will be `True`
- If `--verbose` is passed, `include_workspace_size` will be `True` as well.
- If none is passed, `include_workspace_size` will be `False` as `verbose` will be `False`.

Reply:
Thanks for the catch. Fixed 👍